app/i18n/locales/de.js

```diff
@@ -318,7 +320,7 @@ export default {
 	Message_Reported: 'Nachricht gemeldet',
 	Microphone_Permission_Message: 'Rocket.Chat benötigt Zugriff auf das Mikrofon, damit du eine Audionachricht senden kannst.',
 	Microphone_Permission: 'Mikrofonberechtigung',
-	Mute: 'Stumm',
+	Mute: 'Diesem Benutzer das Chatten verbieten',
```

Comment:
@phriedrich  Are you sure about this one?
It looks so verbose even for german :)
I wonder if it's going to be wrapped or cutted.
Does the previous version work?

Reply:
Yes, this one looks especially cumbersome, but for the place where it is used, this is the actual term from the browser/server app. 

It's in a room's member list, please see the attached screenshots:

![Browser-App](https://user-images.githubusercontent.com/470153/105050545-2d6db900-5a6e-11eb-818c-7ac87a698d2c.png)

![ReactNative](https://user-images.githubusercontent.com/470153/105050558-32326d00-5a6e-11eb-9fc0-24fe01f75612.png)

A shorter version would work, something like "Stumm schalten". 
But this one is more descriptive, and I tried to match the translations of the browser app, where possible. In this case, there's enough space, so I used the original, very verbose, form.

Reply:
Got it. You're right.